posthog/session_recordings/queries/test/__snapshots__/test_session_recording_list_from_filters.ambr

```diff
@@ -165,19 +165,26 @@
                                                                                                                                                                                                                                                                                                                                                                                                                          FROM events
                                                                                                                                                                                                                                                                                                                                                                                                                          WHERE and(equals(events.team_id, 2), notEmpty(events.`$session_id`), greaterOrEquals(toTimeZone(events.timestamp, 'UTC'), toDateTime64('2020-12-11 13:46:23.000000', 6, 'UTC')), lessOrEquals(toTimeZone(events.timestamp, 'UTC'), now64(6, 'UTC')), greaterOrEquals(toTimeZone(events.timestamp, 'UTC'), toDateTime64('2020-12-21 23:58:00.000000', 6, 'UTC')), lessOrEquals(toTimeZone(events.timestamp, 'UTC'), toDateTime64('2021-01-04 23:59:59.999999', 6, 'UTC')), or(and(equals(events.event, '$pageview'), true), and(equals(events.event, 'custom-event'), true)))
                                                                                                                                                                                                                                                                                                                                                                                                                          GROUP BY events.`$session_id`
-                                                                                                                                                                                                                                                                                                                                                                                                                         HAVING hasAll(groupUniqArray(events.event), ['$pageview', 'custom-event']))), true, ifNull(equals(s__pdi.person_id, '00000000-0000-0000-0000-000000000000'), 0))
+                                                                                                                                                                                                                                                                                                                                                                                                                         HAVING hasAll(groupUniqArray(events.event), ['$pageview', 'custom-event']))), in(s.distinct_id,
```

Comment:
oh... this is in HAVING...

that's unexpected and unnecessary

i might fix this in follow-up rather than in one PR 🤔